deps/mbedtls.cmake

```diff
@@ -0,0 +1,31 @@
+# Patch mbedtls
+set(OC_REAPPLY_MBEDTLS_PATCHES ON CACHE BOOL "")
+if(OC_REAPPLY_MBEDTLS_PATCHES)
+    include(${PROJECT_SOURCE_DIR}/deps/mbedtls-patch.cmake)
+    set(OC_REAPPLY_MBEDTLS_PATCHES OFF CACHE BOOL
+        "By default, mbedTLS patches are applied upon the first CMake Configure. Set this to ON to reapply the patches on the next configure."
+         FORCE
+    )
+endif()
+
+file(GLOB MBEDTLS_SRC
+    ${PROJECT_SOURCE_DIR}/deps/mbedtls/library/[a-l]*.c
+    ${PROJECT_SOURCE_DIR}/deps/mbedtls/library/md*.c
+    ${PROJECT_SOURCE_DIR}/deps/mbedtls/library/[n-x]*.c
```

Comment:
As I'm looking at the list of files in `deps/mbedtls/library` isn't this skipping only the `memory_buffer_alloc.c` file? Seems simpler to glob `*.c` here and then add the file to `list(REMOVE_ITEM MBEDTLS_SRC)` call below.

Reply:
You're right. Fixed in https://github.com/iotivity/iotivity-lite/pull/181/commits/7501f8b0c88911708bd389345ea2635c9dff63b5